topdown/crypto.go

```diff
@@ -41,3 +42,67 @@ func builtinCryptoX509ParseCertificates(a ast.Value) (ast.Value, error) {
 func init() {
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificates.Name, builtinCryptoX509ParseCertificates)
 }
+
+// createRootCAs creates a new Cert Pool from scratch or adds to a copy of System Certs
+func createRootCAs(tlsCaCertFile string, tlsCaCertEnvVar []byte, tlsUseSystemCerts bool) (*x509.CertPool, error) {
+
+	var newRootCAs *x509.CertPool
+
+	if tlsUseSystemCerts {
+		systemCertPool, err := x509.SystemCertPool()
+		if systemCertPool == nil {
+			return nil, err
+		}
+		newRootCAs = systemCertPool
+	} else {
+		newRootCAs = x509.NewCertPool()
+	}
+
+	if len(tlsCaCertFile) > 0 {
+		// Append our cert to the system pool
+		caCert, err := ReadCertFromFile(tlsCaCertFile)
+		if err != nil {
+			return nil, err
+		}
+		if ok := newRootCAs.AppendCertsFromPEM(caCert); !ok {
+			return nil, err
+		}
+	}
+
+	if len(tlsCaCertEnvVar) > 0 {
+		// Append our cert to the system pool
+		if ok := newRootCAs.AppendCertsFromPEM(tlsCaCertEnvVar); !ok {
+			return nil, fmt.Errorf("error appending cert from Env Var into System Certs")
+		}
+	}
+
+	return newRootCAs, nil
+}
+
+// ReadCertFromFile reads a cert from file
+func ReadCertFromFile(localCertFile string) ([]byte, error) {
+	// Read in the cert file
+	certPEM, err := ioutil.ReadFile(localCertFile)
+	if err != nil {
+		return nil, err
+	}
+	/*	block, _ := pem.Decode([]byte(certPEM))
+		if block == nil {
+			panic("failed to parse certificate PEM")
+		}
+		cert, err := x509.ParseCertificate(block.Bytes)
+		if err != nil {
+			panic("failed to parse certificate: " + err.Error())
+		}*/
+	return certPEM, nil
+}
+
+// ReadKeyFromFile reads a key from file
+func ReadKeyFromFile(localKeyFile string) ([]byte, error) {
+	// Read in the cert file
```

Comment:
This comment doesn't seem accurate. Also, this could be
```go
func ReadKeyFromFile(localKeyFile string) ([]byte, error) {
 	return ioutil.ReadFile(localKeyFile)
}
```
To be explicit, I'm not saying that these functions don't serve a purpose! After all, calling `ReadKeyFromFile` is so much more expressive than a straightforward `ioutil.ReadFile`.

Reply:
@repenno same comment as above, it looks like these aren't needed.